app/src/main/java/com/github/sdp/tarjetakuna/model/MagicCard.kt

```diff
@@ -98,4 +101,14 @@ data class MagicCard(
             require(subtype.isNotBlank()) { "Subtype cannot be blank" }
         }
     }
+
+    /**
+     * Converts a MagicCard to a MagicCardEntity.
+     */
+    fun toDBMagicCard(possession: CardPossession = CardPossession.NONE): DBMagicCard {
```

Comment:
You should instead create a factory pattern in MagicCardEntity called for example : MagicCardEntity.fromMagicCard(...)